### PR TITLE
Create nb.json

### PR DESCRIPTION
**Pull request by hwikene**
Translation for Norwegian

### CODE REVIEW

**Comment by KoljaWindeler** [comment]
Thanks, can I contact you when I add feature and need translation? 
Kolja

**Comment by hwikene** [comment]
Hi! Yes you can. i`ll be glad to help :)

**Comment by lb3ne** [comment]
@hwikene - Is it rude to correct something in the translation? And, is it possible?

File: custom_components/isc/translations/nb.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                }
+            },
+            "finish": {
+                "title": "ICS",
+                "description": "Skriv inn sensornavnet og konfigurer sensorparametrene. Mer info på https://github.com/koljawindeler/ics#configuration-options.\n Side 2/2",
+                "data": {
+                    "show_blank": "Vis hendelser uten tittel",
+                    "force_update": "Tving periodiske oppdateringer",
+                    "show_remaining": "Vis gjenværende dager",
+                    "show_ongoing": "Vis hendelser som pågå",
```
Comment (lb3ne) [inline custom_components/isc/translations/nb.json:25]: "show_ongoing": "Vis hendelser som pågå"
Suggestion: "show_ongoing": "Vis hendelser som pågår"

**Comment by hwikene** [comment]
No problem! Please do :)

**Pull request merged**